cmd/k8srules/rc.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: rules
+  labels:
+    app: rules
+spec:
+  replicas: 1
+  selector:
+    app: rules
+  template:
+    metadata:
+      name: rules
+      labels:
+        app: rules
+    spec:
+      containers:
+        - name: rules
+          image: amalgam8/a8-k8s-rules-controller:latest
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: K8S_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
```

Comment:
See first comment. We need to merge this into controller such that controller switches to this operating mode when provided with K8S env vars/flags